web/src/components/programs/ProgramEditForm.vue

```diff
@@ -74,7 +70,7 @@
           type="submit"
           class="ml-3"
           :loading="isUpdating"
-          :disabled="!isValid"
+          :disabled="!formRef?.isValid"
```

Comment:
I believe that you can do `v-model="isValid"` on the `<v-form>` object, and then could avoid reading complexity of trying to figure out the effect when form ref is undefined.

Generally I try to avoid having more than one layer of conditionals per line of code.